content/millennium/r4/other/binary.md

```diff
@@ -40,24 +40,50 @@ _Implementation Notes_
 
 <%= authorization_types(provider: true, patient: true, system: true) %>
 
-_Implementation Note_
+_Implementation Notes_
 
-The `Binary.read` scope and either the `DiagnosticReport.read` or the `DocumentReference.read` scope is required.
+* For attachments with the `application/pdf` contentType, the `Binary.read` scope and either the `DiagnosticReport.read` or the `DocumentReference.read` scope is required.
+* For attachments with the `text/html` contentType, the `Binary.read` scope and the `DocumentReference.read` scope are required.
 
 ### Headers
 
+Either `DocumentReference.content.attachment.contentType` or `DiagnosticReport.presentedForm.contentType` should be used to set the `Accept` header. An `Accept` header of `application/fhir+json` could be supplied instead, if the JSON Binary resource is desired instead of the raw data.
```

Comment:
For R4 we do not currently support 'DiagnosticReport' i guess this is a good question for our SD on if we would like to call out this resource.

Reply:
@zxeleven I talked this over with @praneethk2094 and @kathypickering  and we all agreed that we should remove reference to DiagnosticReport.presentedForm.contentType since we don't support that resource yet. Thanks.

Reply:
Updated here: 03cae8556e076838801ac31ce1949680ac811e5e